src/ec/packages/ec-component-site-header-harmonised/ecl-site-header-harmonised.html.twig

```diff
@@ -171,125 +171,138 @@
           class="ecl-site-header-harmonised__logo-image"
           src="{{ _logo.src }}"
         />
-      </a>
-      <div class="ecl-site-header-harmonised__action">
-      {% if _login_toggle is not empty and _login_box is not empty %}
-        <div class="ecl-site-header-harmonised__login-container">
-        {% if _logged %}
-          <a
-            class="ecl-link ecl-link--standalone ecl-site-header-harmonised__login-toggle"
-            href="{{ _login_toggle.href_logged }}"
-            aria-controls="{{ _login_box.id }}"
-            data-ecl-login-toggle
-            aria-expanded="false"
-          >
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'general',
-                name: 'logged-in',
-                size: 'm'
-              },
-              extra_classes: 'ecl-site-header-harmonised__icon'
-            } only %}
-             {{- login_toggle.label_logged -}}
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'ui',
-                name: 'corner-arrow',
-                size: 'xs'
-              },
-              extra_classes: 'ecl-site-header-harmonised__login-arrow'
-            } only %}
-          </a>
-          <div
-            id="{{ _login_box.id }}"
-            class="ecl-site-header-harmonised__login-box"
-            data-ecl-login-box
-          >
-            <p class="ecl-site-header-harmonised__login-description">
-              {{- _login_box.description|raw -}}
-            </p>
-            <hr className="ecl-site-header-harmonised__login-separator" />
-            {% include '@ecl-twig/ec-component-link/ecl-link.html.twig' with {
-              link: {
-                label: _login_box.label,
-                path: _login_box.href,
-                type: "standalone"
-              }
-            } only %}
-          </div>
+        {% if group != 'group3' %}
```

Comment:
We have a local variable _group, please use that cause it defines a default and the value for the group parameter might be simply missing.